spec/models/download_spec.rb

```diff
@@ -99,35 +99,53 @@
   describe "authors" do
     let(:work) { Work.new }
     let(:subject) { Download.new(work) }
+    let(:pseud1) { create(:pseud, name: "First", user: create(:user)) }
+    let(:pseud2) { create(:pseud, name: "Second", user: create(:user)) }
 
     it "joins the author names separated by a comma and a space" do
-      allow(subject).to receive(:author_names).and_return(["First (Zeroth)", "Second"])
+      allow(pseud1).to receive(:byline).and_return("First (Zeroth)")
+      allow(pseud2).to receive(:byline).and_return("Second")
+      allow(work).to receive(:pseuds).and_return([pseud1, pseud2])
 
       expect(subject.authors).to eq("First (Zeroth), Second")
     end
 
-    it "transliterates non-ASCII characters" do
-      allow(subject).to receive(:author_names).and_return(["我哥好像被奇怪的人盯上了怎么破"])
+    it "leaves Chinese characters alone" do
```

Comment:
Can you move this under `describe "page_title"` and remove `describe "authors"` entirely?